docs/examples/fd_fitting_basic/index.rst

```diff
@@ -0,0 +1,194 @@
+.. warning::
+    This is beta functionality. While usable, this is a beta-functionality which
+    has not yet been tested in a sufficient number of different scenarios. The API
+    may also still be subject to change.
+
+Twistable Worm-Like-Chain Fitting
+=================================
+
+.. only:: html
+
+    :nbexport:`Download this page as a Jupyter notebook <self>`
+
+First we import the necessary libraries::
+
+    import matplotlib.pyplot as plt
+    from lumicks import pylake
+
+Load the data from disk
+-----------------------
+
+Let's load and plot the data first::
+
+    file = pylake.File('twlc_data//20200430-163932 FD Curve FD_1_control_forw.h5')
+    fd_curve = file.fdcurves["FD_1_control_forw"]
+    fd_curve.plot_scatter()
+
+.. image:: output_9_1.png
+
+Set up a basic model first
+--------------------------
+
+We clearly see that the force starts levelling out at high forces in the data. This
+is a clear sign that to be able to describe this data, we'll need something more complex
+in order to capture this behaviour we see in the data. The twistable worm like chain
+is one such model that can describe the untwisting of DNA at high forces. However, its
+complexity also incurs some challenges.
+
+Parameter optimization always begins from an initial guess, and if this initial guess
+is bad, it can get stuck at an estimated set of parameters that are suboptimal, a
+so-called local optimum. One way to mitigate this, is to start at good initial values.
+
+In this notebook, we fit the low part of the curve first, with a regular worm like
+chain model, and then use those estimates as initial guesses to fit the twistable model.
+
+Let's set up the Odijk model and create the fit::
+
+    m_odijk = pylake.inverted_odijk('DNA').subtract_independent_offset() + pylake.force_offset('DNA')
+    fit_odijk = pylake.FdFit(m_odijk)
+
+Considering that this model only describes the lower part of the curve, we have to
+extract the data that is relevant to us. We can obtain this data from the ``FdCurve``
+as follows::
+
+    force = fd_curve.f.data
+    distance = fd_curve.d.data
+
+We only wish to use the forces below 30, so we filter the data according to this
+requirement::
+
+    mask = force < 30
+    distance = distance[mask]
+    force = force[mask]
+
+Now we are ready to add this data to the fit. Note that it is important to constrain the distance offset, as this
+provides a lot of additional freedom in the model::
+
+    fit_odijk.add_data("Inverted Odijk", force, distance)
+    fit_odijk["DNA/d_offset"].upper_bound = .01
+    fit_odijk["DNA/d_offset"].lower_bound = -.01
```

Comment:
Maybe add the leading `0` since `-.01` looks a bit strange and could distract from the main point: https://herbsutter.com/2020/05/19/of-feedback-and-little-things/

Reply:
Nice read :)

Changed!